deployer/src/ec2/update.rs

```diff
@@ -0,0 +1,143 @@
+//! `update` subcommand for `ec2`
+
+use crate::ec2::{
+    aws::*, deployer_directory, utils::*, Config, Error, InstanceConfig, CREATED_FILE_NAME,
+    DESTROYED_FILE_NAME, MONITORING_NAME, MONITORING_REGION,
+};
+use aws_sdk_ec2::types::Filter;
+use futures::future::try_join_all;
+use std::collections::HashMap;
+use std::fs::File;
+use std::path::PathBuf;
+use tracing::{error, info};
+
+/// Updates the binary and configuration on all binary nodes
+pub async fn update(config_path: &PathBuf) -> Result<(), Error> {
+    // Load config
+    let config: Config = {
+        let config_file = File::open(config_path)?;
+        serde_yaml::from_reader(config_file)?
+    };
+    let tag = &config.tag;
+    info!(tag = tag.as_str(), "loaded configuration");
+
+    // Ensure created file exists
+    let temp_dir = deployer_directory(tag);
+    let created_file = temp_dir.join(CREATED_FILE_NAME);
+    if !created_file.exists() {
+        return Err(Error::DeploymentNotComplete(tag.clone()));
+    }
+
+    // Ensure destroyed file does not exist
+    let destroyed_file = temp_dir.join(DESTROYED_FILE_NAME);
+    if destroyed_file.exists() {
+        return Err(Error::DeploymentAlreadyDestroyed(tag.clone()));
+    }
+
+    // Construct private key path (assumes it exists from create command)
```

Comment:
On mobile so it's hard to tell but is the formatting strange on this line?

Reply:
Looks normal on my side 🤔